src/main/resources/languages/messages_en.yml

```diff
@@ -15,6 +15,12 @@ commands:
     description: Unlock/Reset researches for a player
     reset: '&cYou have reset %player%''s Knowledge'
     reset-target: '&cYour Knowledge has been reset'
+
+  backpack:
```

Comment:
This is actually a duplicate key. You already placed the description under "commands.backpack".